src/test/java/com/google/roomies/data/CommentTest.java

```diff
@@ -0,0 +1,83 @@
+package com.google.roomies;
+
+import static com.google.roomies.CommentRequestParameterNames.COMMENT;
+import static com.google.roomies.CommentRequestParameterNames.TIMESTAMP;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
+
+import com.google.cloud.firestore.FieldValue;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+import com.google.cloud.Timestamp;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import java.util.Optional;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+@RunWith(JUnit4.class)
+public class CommentTest {
+  @Mock HttpServletRequest request;
+  @Mock QueryDocumentSnapshot queryDocumentSnapshotMock;
+
+  private static final String commentMessage = "Test message";
```

Comment:
Class constants should be all uppercase with words separated by underscores ("_").  ex. COMMENT_MESSAGE